Source/WebCore/page/Quirks.cpp

```diff
@@ -1735,6 +1735,11 @@ bool Quirks::shouldAvoidStartingSelectionOnMouseDown(const Node& target) const
     return false;
 }
 
+bool Quirks::shouldReuseLiveRangeForSelectionUpdate() const
+{
+    return needsQuirks() && isDomain("scribd.com"_s);
```

Comment:
Perhaps it's worth adding a cached bit for this on `QuirksData`, like some of the other quirks? (Since the codepath that calls into this can be invoked quite frequently)

Reply:
Yeah, we can do that. I was thinking of doing it anyway.